CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## [v4.15.1](https://github.com/plivo/plivo-php/releases/tag/v4.15.2) - 2020-11-26
```

Comment:
Since this isn't a bug fix and the API is changing in a backwards compatible way, the minor version can be bumped up to `4.16.0`